### DISCUSSION TITLE
Searchbar not working

### DISCUSSION THREAD

**Discussion by ali4zimi** [nuxt/content#2092] [General]
I have noticed that the website's search bar is not working recently. I can't search for content. However, when I forked it and test it locally, it worked, but I don't know what is wrong. 

Please fix it
![NuxtContent](https://github.com/nuxt/content/assets/13467999/79d7e872-ef53-4b4a-916d-1d08e062464b)

**Comment by nobkd** [comment]
This works again, since the package updates in #2078

It was probably related to #2094
with this solution https://github.com/nuxt-modules/algolia/issues/152#issuecomment-1576642221
released in https://github.com/nuxt-themes/docus/releases/tag/v1.12.3

**Comment by ali4zimi** [comment]
Thanks @nobkd
